tests/typecheck/apps/test_config.yml

```diff
@@ -0,0 +1,24 @@
+-   case: test_appconfig_can_be_str_or_property
```

Comment:
I'd prefer these to be three separate tests that can fail independently :)

Reply:
I've just tried locally and also cannot reproduce the issue. I wonder if there's some Mypy config?

Reply:
Ah the problem was the tests used `django.app.config` instead of `django.apps.config`. Enabling strict mode temporarily in the tests uncovered this, because it showed:

```
E     main:4: error: Class cannot subclass "AppConfig" (has type "Any") (diff)
```

I've just pushed a commit fixing the import path, and the calls, but the tests now fail in other ways.

Reply:
> I'd prefer these to be three separate tests

We have a technical limitation here that each individual test case adds significant time overhead. The test suite takes 11-15 minutes, which is already way too long.

But I think anything except the first test here is a vanishingly small use case and doesn't really need to be tested.

> that can fail independently

Not really a concern for stub tests IMO because one test case can report multiple mypy errors.

Reply:
Okay totally fair, let’s combine them back.

Reply:
Merged the tests.